src/components/FilterInput/index.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import styled from 'styled-components';
+import TextField from '@material-ui/core/TextField';
+
+const Container = styled.div``;
+const TextInput = styled(TextField)`
+  margin-top: 17px !important;
+  margin-left: 10px !important;
+  color: #9b9b9b;
+  font-size: 18px;
+  font-weight: normal;
+  height: 17px;
+  letter-spacing: 0;
+  line-height: 17px;
+  width: 150px;
+`;
+
+interface Props {
+  inputProps?: object;
+  InputProps?: object;
+  placeholder?: string;
+}
+
+class FilterInput extends React.Component<Props> {
```

Comment:
Please use the functional component.

Reply:
in the next ticket this component will start utilizing state